apps/intellij-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -25,9 +25,6 @@
     <!-- Extension points defined by the plugin.
          Read more: https://plugins.jetbrains.com/docs/intellij/plugin-extension-points.html -->
     <extensions defaultExtensionNs="com.intellij">
-        <treeStructureProvider implementation="ai.hash.rustextra.projectView.ModuleGroupTreeStructureProvider" order="last, after org.rust.lang"/>
+        <treeStructureProvider implementation="ai.hash.rustextra.projectView.ModuleGroupTreeStructureProvider" order="last, before org.rust.lang"/>
```

Comment:
Why this dependency is needed at all?

Reply:
```suggestion
        <treeStructureProvider implementation="ai.hash.rustextra.projectView.ModuleGroupTreeStructureProvider" />
```

no longer needed, this was something accidentally left behind from the previous approach